### PR TITLE
Story/us 024 create auth system

### PR DESCRIPTION
**Pull request by JamesMatchett**

### CODE REVIEW

File: kjourney/src/app/login/login.component.css [hunk 1]
```diff
@@ -93,4 +93,39 @@
...
+		font-family: sans-serif;
+		text-align: center;
+		color: white;
+		background-color: rgba(97, 168, 63, 0);
+		border: 2px solid #00000000;
+		
+	}
+
+	.btn btn-primary{
+		background-color:  rgba(66, 103, 159, 0.9);
```
Comment (jessicamcmillen) [inline kjourney/src/app/login/login.component.css]: Change button colour to #283583 - Kainos colour

File: kjourney/src/app/login/login.component.css [hunk 2]
```diff
@@ -93,4 +93,39 @@
 	padding: 0px;
 	text-align: center;
 	color: white;
-  }
\ No newline at end of file
+	}
+	
+	.card{
+		border: 2px solid #00000088;
```
Comment (jessicamcmillen) [inline kjourney/src/app/login/login.component.css:99]: Change border colour to #283583 - Kainos colour

**Review by jessicamcmillen — CHANGES_REQUESTED** [review]

File: kjourney/src/app/app.component.html [hunk 3]
```diff
@@ -14,6 +14,7 @@ <h1></h1>
```
Comment (ShaneKainos) [inline kjourney/src/app/app.component.html:14]: The source for the logo should be changed from src="logo.png" to src="assets/logo.png"

**Review by jessicamcmillen — APPROVED** [review]

**Review by ShaneKainos — COMMENTED** [review]
Image source was changed, everything else seems fine.

**Pull request merged**